lib/teaspoon/driver/selenium.rb

```diff
@@ -23,7 +23,7 @@ def initialize(options = nil)
       end
 
       def run_specs(runner, url)
-        driver = ::Selenium::WebDriver.for(driver_options[:client_driver])
+        driver = ::Selenium::WebDriver.for(driver_options[:client_driver], @options[:client_driver_opts]&.dup || {})
```

Comment:
Doesn't this also make a dependency on whatever Ruby version introduced this new syntax? I think 2.3?